modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -681,6 +681,15 @@
 				[b]Note:[/b] As annotations describe their subject, the [annotation @tool] annotation must be placed before the class definition and inheritance.
 			</description>
 		</annotation>
+		<annotation name="@uid">
+			<return type="void" />
+			<param index="0" name="uid" type="String" />
+			<description>
+				[i]This annotation exists for forward compatibility with 4.3 and beyond. It has no functionality in version 4.2 and below.[/i]
+				Stores information about UID of this script. This annotation is auto-generated when saving the script and must not be modified manually. Only applies to scripts saved as separate files (i.e. not built-in).
+				[b]Note:[/b] Unlike most other annotations, the argument of the [annotation @uid] annotation must be a string literal (constant expressions are not supported).
```

Comment:
This is not true for 4.2 and below, but I think the reader will realize that the information is for 4.3.

Reply:
Hence the note at line 688.

Reply:
Striking through would be a bit too much :) I see the point of removing the note, but at the same time I think that it at least explains how this particular annotations must be formatted. Even if the statement is not exactly true in general.